_docs/tasks/traffic-management-v1alpha3/egress-tcp.md

```diff
@@ -44,16 +44,17 @@ metadata:
   name: wikipedia-ext
 spec:
   hosts:
-  - 91.198.174.192/27
-  - 103.102.166.224/27
-  - 198.35.26.96/27
-  - 208.80.153.224/27
-  - 208.80.154.224/27
+  - wikipedia.org
+  addresses:
+    - 91.198.174.192/27
+    - 103.102.166.224/27
+    - 198.35.26.96/27
+    - 208.80.153.224/27
+    - 208.80.154.224/27
   ports:
-  - number: 443
-    name: tcp
-    protocol: TCP
-  discovery: NONE
```

Comment:
this should be resolution: NONE for clarity

Reply:
fixed